Cargo.toml

```diff
@@ -10,18 +10,29 @@ repository = "https://github.com/sgoudham/uwuifyy"
 keywords = ["cli", "uwu", "owo", "uwuify", "anime"]
 categories = ["command-line-utilities"]
 exclude = [
-    "examples/the-complete-works-of-william-shakespeare.txt",
-    "examples/tiktok_app_reviews.csv",
-    "examples/tokyo-2020-olympics-tweets.csv",
-    "examples/uwu/**"
+  "examples/the-complete-works-of-william-shakespeare.txt",
+  "examples/tiktok_app_reviews.csv",
+  "examples/tokyo-2020-olympics-tweets.csv",
+  "examples/uwu/**",
 ]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[lib]
+name = "uwuify"
```

Comment:
Probably should keep the lib name consistent with the binary name no?

Reply:
You mean by renaming it to `UwUify`? Doing that would produce a lint from the compiler asking us to rename our lib to use snake case.

Reply:
Oh no sorry, I just meant keeping it as `uwuifyy` so people aren't confused? There's unfortunately already a crate called `uwuify` which is the reasoning for the name

Reply:
Oops, fixed that